src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -327,22 +358,30 @@ object DockerPlugin extends AutoPlugin {
     Cmd("COPY", s"$files /$files")
   }
 
+  private final def makeCopy(src: String, dest: String): CmdLike =
```

Comment:
This happens to overload `makeCopy` above, although the behavior differs quite a bit. I think those functions should have different names to avoid confusion. We should probably rename the original one, since it's far less intuitive.